heroku/cli
# refactor(pg-v5): Move command `pg:credentials:repair-default` to oclif

### Description
https://gus.lightning.force.com/lightning/_classic/%2Fa07EE00001bCeHfYAK

This converts `pg:credentials:repair-default` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Run `./bin/run pg:credentials:repair-defualt DB_NAME -a APP` with an app that has a pg db 
3) Verify the command succeeds, restoring the default cred
4) Compare with prod
